certified-connectors/BulkSMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,137 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "BulkSMS",
+    "description": "SMS enable your workflows and automations with the Power Automate Connector for BulkSMS",
```

Comment:
@dan-perrin please remove "Power Automate" from the description. please follow our documentations [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector). I take full responsibility for this mistake I did not see it from before.

Reply:
Thanks Amjed. All good, we have updated the description.